charts/zookeeper-operator/templates/operator.yaml

```
@@ -21,6 +21,10 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
+      {{- if .Values.additionalSidecarVolumes }}
+      volumes:
+{{- include "chart.additionalSidecarVolumes" . | indent 6 }}
```

You can pass `.Values.additionalSidecarVolumes` to the `chart.additionalSidecarVolumes` template or use with to change the 'dot' 
```suggestion
      {{- with .Values.additionalSidecarVolumes }}
      volumes:
{{- include "chart.additionalSidecarVolumes" . | indent 6 }}
```

---

no longer needed

---

no longer needed